src/metabase/models/serialization.clj

```diff
@@ -1223,7 +1223,7 @@
   ported by [[export-viz-click-behavior-mapping]]."
   [mappings]
   (into {} (for [[kw-key mapping] mappings
-                 :let [k (name kw-key)]]
+                 :let [k (u/qualified-name kw-key)]]
```

Comment:
Reading this usage of `u/qualified-name` without the test is pretty confusing - why would a mapping keyword have a namespace? To save devs time in the future puzzling over this, I would leave a comment

Reply:
I don't see a need: you (should) know a mapping keyword shouldn't been a keyword in the first place, it's just how it's processed after being selected from db. In an ideal world we'd either have different data layout for `click_behavior` or not convert it's keys to a keywords.

And `u/qualified-name` just says "give me all content".